### PR TITLE
[muparser] update to 2.3.3-1

### PR DESCRIPTION
**Pull request by MehdiChinoune**
**Describe the pull request**

- #### What does your PR fix?  
  Update muparser to 2.3.3-1

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all
  No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by MehdiChinoune** [comment]
timeout?

File: ports/muparser/portfile.cmake [hunk 1]
```diff
@@ -3,25 +3,27 @@ vcpkg_fail_port_install(ON_TARGET "uwp")
...
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 
-file(INSTALL ${SOURCE_PATH}/License.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
 vcpkg_fixup_pkgconfig()
+
+vcpkg_cmake_config_fixup(
+    PACKAGE_NAME muparser
+    CONFIG_PATH "lib/cmake/muparser"
+)
```
Comment (JonLiu1993) [inline ports/muparser/portfile.cmake:29]:
```suggestion
```

File: ports/muparser/portfile.cmake [hunk 2]
```diff
@@ -3,25 +3,27 @@ vcpkg_fail_port_install(ON_TARGET "uwp")
...
+    OPTIONS
         -DENABLE_SAMPLES=OFF
         -DENABLE_OPENMP=OFF
-    OPTIONS_DEBUG 
-        -DDISABLE_INSTALL_HEADERS=ON
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 vcpkg_copy_pdbs()
```
Comment (JonLiu1993) [inline ports/muparser/portfile.cmake:20]:
```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/muparser")
```

File: ports/muparser/portfile.cmake [hunk 3]
```diff
@@ -3,25 +3,27 @@ vcpkg_fail_port_install(ON_TARGET "uwp")
...
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 
-file(INSTALL ${SOURCE_PATH}/License.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
 vcpkg_fixup_pkgconfig()
+
+vcpkg_cmake_config_fixup(
+    PACKAGE_NAME muparser
+    CONFIG_PATH "lib/cmake/muparser"
```
Comment (JonLiu1993) [inline ports/muparser/portfile.cmake:28]:
```suggestion
    CONFIG_PATH lib/cmake/muparser
```

File: ports/muparser/vcpkg.json [hunk 4]
```diff
@@ -1,8 +1,18 @@
 {
   "name": "muparser",
-  "version-string": "2.3.2",
-  "port-version": 2,
+  "version-string": "2.3.3-1",
```
Comment (JonLiu1993) [inline ports/muparser/vcpkg.json:3]:
```suggestion
  "version": "2.3.3-1",
```

**Comment by MehdiChinoune** [comment]
done!

**Review by JonLiu1993 — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Thanks a bunch for the update PR!

**Pull request merged**